DOCS/interface-changes.rst

```diff
@@ -61,6 +61,16 @@ Interface changes
       `--gamma` to float.
     - add `platform` property
     - add `--auto-window-resize`
+    - `--save-position-on-quit` and its associated commands now store state files in
+      the XDG_STATE_HOME directory by default. This only has an effect on linux/bsd
+      systems.
+    - add `--icc-cache` and `--gpu-shader-cache` options
```

Comment:
If we have never released these new cache options, then we should at the very least remove them from all documentation, and arguably remove them completely rather than ship them in a deprecated state.

Reply:
What do you mean? Don't have these two options and always enable cache?

Reply:
Oh wait, I see, `--icc-cache` used to exist as a option hmm.